### PR TITLE
improve documentation readability

### PR DESCRIPTION
**Pull request by simbabque**

### CODE REVIEW

File: lib/LWP/ConnCache.pm [hunk 1]
```diff
@@ -245,7 +245,7 @@ that can be cached.  The first parameter is a short string like
 Drop connections by some criteria.  The $checker argument is a
 subroutine that is called for each connection.  If the routine returns
 a TRUE value then the connection is dropped.  The routine is called
-with ($conn, $type, $key, $deposit_time) as arguments.
+with (C<$conn>, C<$type>, C<$key>, C<$deposit_time>) as arguments.
 
 Shortcuts: If the $checker argument is absent (or C<undef>) all cached
```
Comment (vanHoesel) [inline lib/LWP/ConnCache.pm:250]: ☝️

Comment (simbabque) [reply lib/LWP/ConnCache.pm]: I'm not sure I understand what you mean, but I've made the whole list one code element now.

**Review by oalders — APPROVED** [review]
Thanks, @simbabque!

File: lib/LWP/ConnCache.pm (hunk 1 above)
Comment (vanHoesel) [reply lib/LWP/ConnCache.pm]: it was about `$checker`

**Pull request merged**